docs/management/advanced-options.asciidoc

```diff
@@ -478,14 +510,6 @@ of buckets to try to represent.
 [[kibana-visualization-settings]]
 ==== Visualization
```

Comment:
This section uses a combination of a table and and table.  Should be consistent.